src/test/java/learn/field_agent/domain/AliasServiceTest.java

```diff
@@ -72,6 +72,9 @@ void shouldNotAddDuplicateWithNullPersona() {
         Alias aliasIn = new Alias(0, "Arianna Martell", null, 4);
         Alias aliasOut = new Alias(5, "Arianna Martell", null, 4);
 
+        when(repository.findAll()).thenReturn(List.of(
+                new Alias(1, "Arianna Martell", null, 1)
+        ));
```

Comment:
The repository's `findAll()` method was being called but since you weren't mocking it, the mock instance was just returning an empty list.